pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -213,6 +216,23 @@ func validateImmutableFieldsCluster(new, old *Cluster) field.ErrorList {
 			field.Forbidden(specPath.Child("ControlPlaneConfiguration"), fmt.Sprintf("field is immutable %v", new.Spec.ControlPlaneConfiguration)))
 	}
 
+	if old.Spec.BundlesRef != nil && new.Spec.BundlesRef == nil {
+		allErrs = append(
+			allErrs,
+			field.Forbidden(specPath.Child("BundlesRef"), fmt.Sprintf("field cannot be removed after setting. Previous value %v", old.Spec.BundlesRef)))
+	}
+
+	standardBundleRef := regexp.MustCompile(`^bundles-[0-9]+$`)
```

Comment:
`MustCompile` will panic, which will crash the controller
If we want to construct the regex in the method scope, we should use the regexp method that returns an error
If not, I would vote to put this in a package private var